client/src/pages/Profile.js

```diff
@@ -101,9 +101,19 @@ const useStyles = makeStyles(theme => ({
 
 const Profile = ({ location }) => {
     const style = useStyles();
+    const username = location.pathname.split('/')[2];
+    
     let [activePanel, toggle] = useState('board');
+    let [boards, setBoards] = useState(['board1','board2','board3'])
+    let [posts, setPosts] = useState(['post1','post2'])
+    
+    useEffect(() => {
+        fetch(`users/${username}`)
+        .then(response => response.json())
+        .then(data => setBoards((boards = data['user']['boards'])))
+        .then(data => setPosts((posts = data['user']['posts'])))
```

Comment:
You don't need to do:
`setBoards((boards = data['user']['boards']))`
you can do:
`setBoards(data['user']['boards'])`

Reply:
okay